### PR TITLE
Add command errors

### PR DESCRIPTION
**Pull request by aljones15**
Just some little helpers for future users.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
...
   .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
   .option('-g, --getRecordId <value>', 'Get a record by ID.')
 );
 
 bedrock.events.on('bedrock.started', async () => {
   try {
     await work();
   } catch(e) {
+    console.error(
```
Comment (mattcollier) [inline index.js:32]: just add this text to the front of the error below

File: index.js [hunk 2]
```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
   .option('-h, --hostname <value>', 'The ledger host.')
   .option('-k, --ignoreSslErrors', 'Use strict SSL.')
   .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
```
Comment (mattcollier) [inline index.js]: would `recordJson` or `recordFile` be better?

File: index.js [hunk 3]
```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
...
   } catch(e) {
+    console.error(
+      'event bedrock.started failed when trying to initialize work.');
     console.error(e.message);
   }
   bedrock.exit();
 });
 
+// gets or creates a default record
+function getRecord(path) {
```
Comment (mattcollier) [inline index.js]: prefix with underscore like all the other private functions in this module.  Also, I see that there is a naming conflict.  Maybe this is `_getTestRecord`?

File: index.js [hunk 4]
```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
...
+  return {
+    '@context': constants.TEST_CONTEXT_V1_URL,
+    id: `https://example.com/transaction/${uuid()}`,
+    price: Math.floor(Math.random() * 1000000000000),
+  };
+}
+
 async function work() {
-  const {getRecordId, hostname, ignoreSslErrors, send} = bedrock.program;
+  const {getRecordId, recordPath,
```
Comment (mattcollier) [inline index.js]: fix style, probably line break at the equal sign.

File: index.js [hunk 5]
```diff
@@ -41,33 +58,41 @@ async function work() {
...
+      '@context': constants.WEB_LEDGER_CONTEXT_V1_URL,
+      creator,
+      type: 'CreateWebLedgerRecord',
+      record
+    };
+    await ledgerClient.sendOperation({operation});
+    console.log(
+      `Operation sent successfully.\n${JSON.stringify(operation, null, 2)}`);
+  } catch(e) {
+    console.error('command send record failed.');
```
Comment (mattcollier) [inline index.js]: combine with logging immediately below.

File: README.md [hunk 6]
```diff
@@ -1 +1,31 @@
 # bedrock-ledger-test-cli
+
+Setup:
```
Comment (mattcollier) [inline README.md]: This should be a heading ## or ###

**Review by mattcollier — CHANGES_REQUESTED** [review]
Thanks for the improvements.

File: index.js (hunk 2 above)
Comment (aljones15) [reply index.js]: file might be good Json I think implies we passing a string that can be json.

File: README.md (hunk 6 above)
Comment (aljones15) [reply README.md]: Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

File: index.js (hunk 5 above)
Comment (aljones15) [reply index.js]: Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

File: index.js (hunk 4 above)
Comment (aljones15) [reply index.js]: Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

File: index.js (hunk 3 above)
Comment (aljones15) [reply index.js]: Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

File: index.js (hunk 2 above)
Comment (aljones15) [reply index.js]: addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

File: index.js (hunk 1 above)
Comment (aljones15) [reply index.js:32]: Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd

**Comment by aljones15** [comment]
@mattcollier updated this PR do to time issues i.e. changes to other projects if you need further changes just make them yourself if possible.

File: README.md (hunk 6 above)
Comment (mattcollier) [reply README.md]: My comment was in reference to L3 'Setup:' and the patch changes the number of pound signs on L1.

File: README.md [hunk 7]
```diff
@@ -1 +1,31 @@
...
+cd bedrock-ledger-test-cli
+npm run send -- -h your-ledger-node:port -k
+```
+
+cli commands
+```
+  .option('-h, --hostname <value>', 'The ledger host.')
+  .option('-k, --ignoreSslErrors', 'Use strict SSL.')
+  .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
```
Comment (mattcollier) [inline README.md]: I guess this needs to be updated to match the options/descriptions that are now in the code.

File: README.md [hunk 8]
```diff
@@ -1 +1,31 @@
 # bedrock-ledger-test-cli
+
+## Setup:
+
+  - You need to have a [web ledger node](https://github.com/digitalbazaar/bedrock-ledger-app-template/tree/initial) already running somewhere.
```
Comment (mattcollier) [inline README.md]: The URL you have linked to here is not going to be valid after that gets merged to master.  Maybe take off the branch info and link just to the repo (even though there's nothing on master yet).

**Review by mattcollier — CHANGES_REQUESTED** [review]
Almost there.  Thanks.  :)

File: README.md (hunk 8 above)
Comment (aljones15) [reply README.md]: as much as I do hate to say this I do think this one is relative as linking to an actually working branch is probably better than master which does not contain working code. If a developer runs into an empty branch I think they can infer which one to use easier than say master. change made though.

Comment (aljones15) [reply README.md]: on the other hand I guess master is better in the long term. way to think about our future.

File: README.md (hunk 7 above)
Comment (aljones15) [reply README.md]: updated.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**